gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java

```diff
@@ -32,14 +31,19 @@
 import org.apache.orc.mapreduce.OrcMapreduceRecordWriter;
 import org.apache.orc.mapreduce.OrcOutputFormat;
 
-import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.*;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter;
+import org.apache.gobblin.writer.GobblinOrcWriter;
+
+import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.COMPACTION_OUTPUT_EXTENSION;
```

Comment:
This doesn't seem to be needed.

Reply:
It is needed in the getRecordWriter() method. Previous import was: 
import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.*; which I changed to
import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.COMPACTION_OUTPUT_EXTENSION;

Let me know, if you want to preserve the original import statement.

Reply:
no, explicit import is obviously better here. thanks for fixing this.